### ISSUE TITLE
A serious mistake

### ISSUE THREAD

**Issue by 332lfgb6** [fomantic/Fomantic-UI#2546]
## nov 11 2022 11:50
![image](https://user-images.githubusercontent.com/51245335/200351817-eb54c5d6-d639-4c8b-a9ba-da1b45bf8ecf.png)

**Comment by lubber-de** [comment]
What exactly do you declare as "serious mistake"? The input field not being wide enough?

**Comment by 332lfgb6** [comment]
> What exactly do you declare as "serious mistake"? The input field not being wide enough?

yes....I feel that all English should be shown

**Comment by 332lfgb6** [comment]
> What exactly do you declare as "serious mistake"? The input field not being wide enough?

I thought you didn't test this place, so just to remind you, did you design it that way?

**Comment by lubber-de** [comment]
> I thought you didn't test this place, so just to remind you, did you design it that way?

No, the input field uses the browser default width. of course users have to adjust the width according to their needs and expected output format. Moving to docs repo, will fix it there
